### ISSUE TITLE
Statsd epoll warn logs are verbose when statsd down

### ISSUE THREAD

**Issue by shakuzen** [micrometer-metrics/micrometer#2624]
Testing on Linux, I can see we get logs like the following with this change (and therefore epoll enabled) when the statsd daemon isn't running:
```
2021-05-28 03:11:45.544  WARN 6373 --- [    udp-epoll-1] i.m.s.reactor.netty.channel.FluxReceive  : [id:35f314ec, L:/127.0.0.1:60434 - R:localhost/127.0.0.1:8125] An exception has been observed post termination, use DEBUG level to see the full stack: java.net.PortUnreachableException: readAddress(..) failed: Connection refused
2021-05-28 03:11:45.554  WARN 6373 --- [    udp-epoll-1] i.m.s.reactor.netty.channel.FluxReceive  : [id:35f314ec, L:/127.0.0.1:60434 - R:localhost/127.0.0.1:8125] An exception has been observed post termination, use DEBUG level to see the full stack: java.net.PortUnreachableException: readAddress(..) failed: Connection refused
```
This is a bit verbose compared to without epoll enabled, but we can improve that in a separate change, I think.
I also verified that with epoll enabled and the statsd daemon running, metrics are received and no logs are output, as expected.

_Originally posted by @shakuzen in https://github.com/micrometer-metrics/micrometer/pull/2611#issuecomment-849839027_

**Comment by sureshnomula74** [comment]
@shakuzen  is this issue fixed?

**Comment by shakuzen** [comment]
It has not been fixed (this issue is still open), but note that this issue is only about overly verbose logging, and not a functional issue. If the configured statsd daemon address is reachable, these logs will not be output and your metrics will be sent. If you are experiencing a functional issue, please open a separate issue with details so we can diagnose that.

**Comment by adithyaonline** [comment]
Any updates about this issue ?
We don't use statsd, Is there a way to disable statsd completely to make this error go away ??

**Comment by shakuzen** [comment]
> We don't use statsd, Is there a way to disable statsd completely to make this error go away ??

If you don't use statsd, do not add a dependency on the micrometer-registry-statsd module and the issue mentioned here cannot happen.

**Comment by abhishekladdha13** [comment]
@jonatan-ivanov @shakuzen
 is there any open PR to fix this issue ?

**Comment by shakuzen** [comment]
> is there any open PR to fix this issue ?

No. We would be happy to review such a pull request. The issue is labelled with the `help wanted` label.

**Comment by jon-signal** [comment]
I've made an attempt to fix this in #4042.

**Comment by jon-signal** [comment]
> We would be happy to review such a pull request.

@shakuzen I recognize this may not be a top priority, but I'm hoping to take you up on the offer to review a pull request (specifically #4042). I do think that change will make a big difference with minimal risk.

**Comment by shakuzen** [comment]
Thanks for the reminder. Sorry we didn't review earlier. I've left a review on the PR now.
